ironfish/src/sdk.ts

```diff
@@ -317,6 +317,7 @@ Use 'ironfish config:set' to connect to a node via TCP, TLS, or IPC.\n`)
       ApiNamespace.rpc,
       ApiNamespace.wallet,
       ApiNamespace.worker,
+      ApiNamespace.node,
```

Comment:
This will also expose `node/getStatus` and `node/getLogStream` endpoints

Hitting the `node/getStatus` endpoint will fail because it isn't a full node, but that's probably OK